### PR TITLE
Feature Spec coverage and percy

### PR DESCRIPTION
**Pull request by Br3nda**
# What has changed and why?
More coverage of rebate forms 
Added percy. 

# How to test this change

- [x] has tests
- [x] at least one a reviewer ran the code

### CODE REVIEW

File: spec/features/admin/rebate_forms/show_spec.rb [hunk 1]
```diff
@@ -60,39 +65,33 @@
...
+          it { expect(page).to_not have_text('Signature required') }
+          include_examples 'percy snapshot'
         end
       end
     end
 
-    context 'when the user does have a name' do
-      let(:user) { FactoryBot.create :admin_user, name: 'George Foreman' }
+    context 'when the user has a name' do
+      let(:user) { FactoryBot.create :admin_user, name: 'George Foreman', email: 'frontlinestaff@regionalcouncil.govt.nz' }
```
[the remark this answers is not in the document]
Comment (Br3nda) [reply spec/features/admin/rebate_forms/show_spec.rb]:
```suggestion
      let(:user) do
        FactoryBot.create :admin_user, name: 'George Foreman', email: 'frontlinestaff@regionalcouncil.govt.nz'
      end
```

**Comment by Br3nda** [comment]
I've used timecop to freeze time during the QR generation, in the hopes that'll make it generate the same image each time and not trigger percy to flag visual differences.

File: spec/support/percy.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+RSpec.shared_examples 'percy snapshot' do
+  # Tell faker gem to use the same random fakes every time
+  # So it doesn't change on every percy run
+  before { Faker::Config.random = Random.new(42) }
+  subject(:example_name) do |example|
+    example.metadata[:full_description]
+  end
+  after { Percy.snapshot(page, name: example_name, widths: [1280]) }
+end
```
Comment (Br3nda) [inline spec/support/percy.rb]: I'm feeling really clever with this little file.

**Comment by mischa-s** [comment]
weird @CaraHill  doesn't have the `.reverse` problem, neither it seems does travis... must be something local :confused:

**Pull request merged**